cdp_scrapers/static_data.md

```diff
@@ -0,0 +1,167 @@
+
+static_data = parse_static_file("cdp_scrapers/instances/seattle-static.json")
+pprint.pp(static_data.seats, indent=4)
+
+# {
+#     'Position 1': Seat(name='Position 1',
+#     ...,
+#     'Position 2': Seat(name='Position 2',
+#     ...,
+# }
+
+pprint.pp(static_data.primary_bodies, indent=4)
+
+# {
+#     'City Council': Body(name='City Council',
+#     ...,
+#     'Council Briefing': Body(name='Council Briefing',
+#     ...,
+# }
+
+pprint.pp(static_data.persons, indent=4)
+
+# {
+#     'Alex Pedersen': Person(name='Alex Pedersen',
+#     ...,
+#                             seat=Seat(name='Position 4',
+#     ...,
+#                             roles=[   Role(title='Councilmember',
+#                                            body=Body(name='City Countil',
+#     ...,
+#     'Andrew Lewis': Person(name='Andrew Lewis',
+#     ...,
+# }
+
+# councilmember is cdp_backend.pipeline.ingestion_models.Person
+# Augment, clean up and update scraped list of roles for this councilmember
+councilmember.seat.roles = sanitize_roles(councilmember.name, scraped_roles, static_data)
+```
+
+### `LegistarScraper`
+
+If your scraper inherits from [`LegistarSCraper`](./legistar_utils.py),
+you could pass the return value from the function into the constructor.
+
+```Python
+# instances/my_municipality.py
+
+from ..legistar_utils import LegistarScraper
+from ..scraper_utils import parse_static_file
+
+class MyScraper(LegistarScraper):
+    PYTHON_MUNICIPALITY_SLUG: str = "mymunicipality"
+
+    def __init__(self):
+        super().__init__(
+            client="municipality_legistar_client_id",
+            timezone="America/My_TimeZone",
+            #
+            static_data=parse_static_file("municipality_static_data.json"),
+            #
+            ...,
+```
+
+The information read from the static data file will be used automatically during [post-processing](./legistar_utils.py#L1560) of the scraped data.
+
+## Static Data File
+
+See [`instances/*static*.json`](./instances/seattle-static.json) for examples.
+
+### `seats`
+
+Seats on the local council defined as [`ingestion_models.Seat`](https://councildataproject.org/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Seat) in JSON.
+Used as a look-up table for `Person.seat` in the `persons` section of the static data file.
+
+### `primary_bodies`
+
+The primary council bodies defined as [`ingestion_models.Body`](https://councildataproject.org/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Body) in JSON.
+A councilmember's [roles](https://councildataproject.org/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Seat.roles) in any of the `primary_bodies` are used to determine the person's terms served.
+
+### `persons`
+
+Councilmembers defined as [`ingestion_models.Person`](https://councildataproject.org/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Person) in JSON.
```

Comment:
Rephrease this. It's not _100%_ the `Person` model in JSON (unlike the other `seats` and `primary_bodies` sections of the file).